topdown/crypto.go

```diff
@@ -65,11 +66,40 @@ func builtinCryptoSha256(a ast.Value) (ast.Value, error) {
 	return hashHelper(a, func(s ast.String) string { return fmt.Sprintf("%x", sha256.Sum256([]byte(s))) })
 }
 
+func builtinCryptoX509ParseCertificateRequest(a ast.Value) (ast.Value, error) {
+	str, err := builtinBase64Decode(a)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
I was going to comment that we might want to have this builtin take in the raw PEM format versus requiring it to be base64 encoded, but then realized that we do the same thing for the `parse_certificates` one.. so I guess we should keep them consistent.

We should probably document what the expected input is for these in the policy reference.

Reply:
Added additional documentation to this end.